community/maturity.md

```diff
@@ -0,0 +1,99 @@
+| **CO40** | The community strives to be meritocratic and gives more rights and responsibilities to contributors who, over time, add value to the project.                          | **YES** The community has elected 1 new PPMC members and 9 new committers so far.                                                                                                                                         |
+| **CO50** | The project documents how contributors can earn more rights such as commit access or decision power, and applies these principles consistently.                        | **YES** The community has clear docs on becomming committers and PPMC members                                                                                                                                             |
+| **CO60** | The community operates based on consensus of its members (see CS10) who have decision power. Dictators, benevolent or not, are not welcome in Apache projects.         | **YES** All decisions are made after vote by community members.                                                                                                                                                           |
+| **CO70** | The project strives to answer user questions in a timely manner.                                                                                                       | **YES** We use dev@uniffle.apache.org, [GitHub issue](https://github.com/apache/incubator-uniffle/issues) and [GitHub discussion](https://github.com/apache/incubator-uniffle/discussions) to do this in a timely manner. |
+
+### Consensus
+
+| **ID**   | **Description**                                                                                                                                                                                                           | **Status**                                                                                                 |
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------------------------------|
+| **CS10** | The project maintains a public list of its contributors who have decision power. The project's PPMC (Project Management Committee) consists of those contributors.                                                        | **Yes** See [members](https://uniffle.apache.org/community/project%20committers/) with committers.         |
```

Comment:
As terminology as been defined at the first mention, it is not necessary to explain it here.

Reply:
We should answer every question.